src/Membership/MembershipRequest.php

```diff
@@ -152,6 +167,7 @@ public static function createFromAdherent(Adherent $adherent): self
         $dto->address = Address::createFromAddress($adherent->getPostAddress());
         $dto->phone = $adherent->getPhone();
         $dto->emailAddress = $adherent->getEmailAddress();
+        $dto->mandate = $adherent->getMandate();
```

Comment:
what about: 
```
if ($dto->mandate = $adherent->getMandate()) {
  $this->elected = true;
}
```
it will display the select automaticaly on the edition page?